### ISSUE TITLE
Icon request: Substack

### ISSUE THREAD

**Issue by mvxt** [FortAwesome/Font-Awesome#16586]
**What can it be used for?**
Requesting an icon for [Substack](https://substack.com/), a growing publishing platform for writers to directly connect with their readers.

**Icon request checklist**
- [x] This is a single icon or matched pair (Ex: `lock` / `unlock`)
- [x] The title starts with "Icon request: " and is followed by the requested icon name (Ex: `Icon request: magical-unicorn`)
- [x] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [x] I have included at least one example image if it could be helpful (optional)
- [x] The request is for a concrete object (it's harder to make an icon to represent happiness, it's easier to make a smiley face. ☺)

**Comment by ghosh-r** [comment]
This is the logo.

![substack-logo](https://substack.com/img/substack.png)

**Comment by mrbluecoat** [comment]
Any ETA?

**Comment by baptiste00001** [comment]
Hello
I see that the substack icon is still not available.
Any release date ?

**Comment by bee-san** [comment]
🙏🏻  Please <3

**Comment by Olshansk** [comment]
What are the blockers on this FR?

**Comment by ilyseiris** [comment]
Would also love a Substack icon! Thank you!

**Comment by Olshansk** [comment]
I just submitted a request directly to substack via their online form [here](https://support.substack.com/hc/en-us/requests/new). Hopefully someone on their team, with access to all the assets could submit a PR.

**Comment by Guerillero** [comment]
I would love to use this on my website

**Comment by mimicpublishing** [comment]
Hello folks. It's been like 2 years. Substack is huge. It's one icon. Let's add it please!

**Comment by ziglertron** [comment]
Piling on here - why isn't Substack a Font Awesome icon yet?

**Comment by kasramp** [comment]
Plus one on this issue. It would be great if the Substack icon is added.

**Comment by llsaboya** [comment]
Yes, it is a must-have

**Comment by stephbairey** [comment]
adding +1, really need a substack icon!

**Comment by wravoc** [comment]
What is taking so long for the Substack Icon? Medium relies on Twitter logins which are super low now and it's keeps going lower on the page rankings, 25 places at a time. Whilst Substack has increased it's website ranking by 198 places.

**Comment by inspiredearth** [comment]
+1 — Putting my hand up for this.
@tagliala  — Is there any chance of getting ETA or some movement on this? Astounding it's taken so long for such a popular platform. 🤷‍♂️

**Comment by tagliala** [comment]
>  Is there any chance of getting ETA or some movement on this? 

You may be interested in taking a look at our [Requesting Icons FAQs](https://fontawesome.com/support#requesting-icons), they contain a lot of information about how requests work

**Comment by inspiredearth** [comment]
> > Is there any chance of getting ETA or some movement on this?
> 
> You may be interested in taking a look at our [Requesting Icons FAQs](https://fontawesome.com/support#requesting-icons), they contain a lot of information about how requests work

Thanks @tagliala . So am I correct in understanding that, so far, there simply hasn't been enough interest (i.e., (`+1`, `heart`, `hooray`, etc.) to warrant further attention on this request?

**Comment by perkdrew** [comment]
How are we still waiting for the Substack icon??

**Comment by laletterata** [comment]
Hi... PLEASE put up a Substack logo. For realz. Thanks

**Comment by Olshansk** [comment]
I emailed substack's support, and I personally think that there might be more behind the scenes w.r.t why this isn't happening.

![Screenshot 2023-10-18 at 1 46 55 PM](https://github.com/FortAwesome/Font-Awesome/assets/1892194/ca9ab47b-513b-444c-bbad-ef99b6e9b169)

![Screenshot 2023-10-18 at 1 47 08 PM](https://github.com/FortAwesome/Font-Awesome/assets/1892194/336f7b4c-ca49-4f86-9595-083f5818cdaa)

**Comment by MarcusTheSmith** [comment]
+1

**Comment by mcansky** [comment]
:+1:

**Comment by dabit3** [comment]
ummm +1

**Comment by Telpidus** [comment]
+1 !

**Comment by kaneofawesome** [comment]
+1

**Comment by Dunya-8a** [comment]
+1

**Comment by amitunboxed** [comment]
+1

**Comment by blpdelivery** [comment]
👍

**Comment by interzonestudio** [comment]
+1

**Comment by stephenreid321** [comment]
+1

**Comment by grndrgn** [comment]
+1

**Comment by ghost** [comment]
+1

**Comment by jt5301** [comment]
+1

**Comment by akawamara** [comment]
+1

**Comment by mcPear** [comment]
+1

**Comment by mohd-alwedaei** [comment]
+1

**Comment by inspiredearth** [comment]
@tagliala Is there any particular reason there's been no movement on this in over four years?

**Comment by mrbluecoat** [comment]
https://simpleicons.org/?q=Substack

**Comment by stephbairey** [comment]
Whoa, it happened

**Comment by widiarifki** [comment]
+1

**Comment by intellerati** [comment]
Could you please let us know when we might expect a Substack icon?

**Comment by MikMuller** [comment]
Yes, please! I have customers who use this platform and need the icon in my set!

**Comment by krisfreedain** [comment]
so curious why this has been open for 4 years

**Comment by aqcheng** [comment]
+1

**Comment by slosar** [comment]
+1

**Comment by ClaudiaIV** [comment]
+1

**Comment by Blackdog67** [comment]
+1
Why is it so difficult for someone to add a Substack icon. Substack is now MASSIVELY popular and I really need to add an icon to my social buttons.

**Comment by SillasMC** [comment]
+1
This icon would be very useful for a lot of projects

**Comment by camillareis** [comment]
+1 please!

**Comment by anandwana001** [comment]
+1

**Comment by likejazz** [comment]
+1

**Comment by andrepache-co** [comment]
+1

**Comment by kelseypedersen** [comment]
+1

**Comment by inspiredearth** [comment]
> > Is there any chance of getting ETA or some movement on this?
> 
> You may be interested in taking a look at our [Requesting Icons FAQs](https://fontawesome.com/support#requesting-they contain a lot of information about how requests work


@tagliala would you please help us understand what the issue is with this request?

It now has 109 reactions on the original post and over 50 comments—significantly more activity than any request currently on the [In Progress Leaderboard](https://fontawesome.com/leaderboard/in-progress). It also has more activity than many, if not most, of those on the [Released Leaderboard](https://fontawesome.com/leaderboard/released).

I would appreciate some clarification on how this all works. SubStack is now a major online publishing platform. Surely it qualifies for being added to FontAwesome?

Thank you.

**Comment by aaronfeingold** [comment]
+1

**Comment by inspiredearth** [comment]
Out of interest, what's the reason a lot of `+1` replies have recently been "Hidden as abuse".
Isn't adding a `+1` message fairly typical and expected behaviour in this context?

**Comment by zmartin96** [comment]
4 years later, adding to this request

**Comment by tagliala** [comment]
> Isn't adding a +1 message fairly typical and expected behaviour in this context?

It was several years ago before GitHub reaction were introduced

Then, as per our [contributing guidelines for requesting a new icon](https://fontawesome.com/support#requesting-icons), it is asked to add a thumbs up reaction because they do not spam subscribers and keeps requests small and easy to read.

+1 reactions are counted in the leaderboard https://fontawesome.com/leaderboard/brands

**Comment by benman1** [comment]
There's been no work on this for the last 5 years, it seems. If anyone is wondering how to get the substack icon, there's one in [simpleicons](https://simpleicons.org/). You can either download the icons and embed them as images, or - much easier even - use simpleicons integrations, available in latex, for example.

**Comment by oorkris** [comment]
No progress after 5 years?

**Comment by TomAndThat** [comment]
Also curious to know why this is not yet available ☺️

**Comment by EricPaulson** [comment]
Yet another dev wondering why this simple icon is still not available.

**Comment by Dunya-8a** [comment]
@robmadole Could you please help here? I've been following this issue for 3 years. Font-Awesome is great and it's really frustrating to not find substack and having to implement it manually.

**Comment by jemmapollari** [comment]
+1 on this request.

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No worries.
* This brand has gone the way of the dodo.
